File: Data Structures & Algorithms/is-anagram/submission-0.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        if(s.length() != t.length()){return false;}
        int result[26]={0};
        for(char c : s){
            result[c - 'a']++;
        }
        for(char c : t){
            result[c - 'a']--;
        }
       sort(s.begin(),s.end());
       sort(t.begin(),t.end());
       return s==t;
    }
};

File: Data Structures & Algorithms/last-stone-weight/submission-0.java
class Solution {
    public int lastStoneWeight(int[] stones) {
        int length=stones.length;
        while(length > 1){
            Arrays.sort(stones,0,length);
            int highest = stones[length - 1];
            int secondHighest = stones[length - 2];
            length=length-2;
            if (highest != secondHighest) {
                stones[length] = highest - secondHighest;
                length++;
            }
        }
        return length == 0 ? 0 : stones[0]; 
        
    }
}
